test/benchmark/tests/test_vgg16.py

```diff
@@ -54,21 +55,21 @@ def test_vgg16_benchmark(
 
         docker_repo_tag = f"{DEFAULT_DOCKER_DEV_ECR_REPO}:{docker_tag}"
 
-        if ec2_instance_type[:2] in GPU_INSTANCES and "gpu" in docker_tag:
+        if ec2_instance_type[:2] in GPU_INSTANCES and ("gpu" in docker_tag or "neuron" in docker_tag):
             dockerImageHandler = DockerImageHandler(docker_tag, cuda_version)
             dockerImageHandler.pull_docker_image_from_ecr(
                 account_id, DEFAULT_REGION, docker_repo_tag, connection=ec2_connection
             )
             docker_repo_tag_for_current_instance = docker_repo_tag
             cuda_version_for_instance = cuda_version
             break
-        if ec2_instance_type[:2] not in GPU_INSTANCES and "cpu" in docker_tag:
+        if ec2_instance_type[:2] not in GPU_INSTANCES and ("cpu" in docker_tag or "neuron" in docker_tag):
             dockerImageHandler = DockerImageHandler(docker_tag, cuda_version)
             dockerImageHandler.pull_docker_image_from_ecr(
                 account_id, DEFAULT_REGION, docker_repo_tag, connection=ec2_connection
             )
             docker_repo_tag_for_current_instance = docker_repo_tag
-            cuda_version_for_instance = cuda_version
+            cuda_version_for_instance = None
```

Comment:
Have we validated on GPU/CUDA after these changes?

Reply:
Yes, in fact validation on CPU was failing and hence this 'cuda_version_for_instance' change was a bug fix, the value should have been None. I've validated the GPU docker image is used after these changes.